changelog.d/20231018_224126_andrey_bulk_save_server_files.md

```diff
@@ -0,0 +1,4 @@
+### Fixed
+
+- Race condition for the task data upload request
```

Comment:
It would be useful to explain what problems that race condition caused for the user.